operational/flexera/custom_branding/README.md

```diff
@@ -0,0 +1,75 @@
+# Custom Branding
+
+## What It Does
+
+This Policy allows a customer to brand the FlexeraOne platform quickly and consistently.
+
+## How It Works
+
+- Run the Custom Branding policy in the tenant ensuring you select the credentials required
+- For non MSP Customers you can only apply the All Customer Settings Category
+- For MSP Customers you can apply bother All Customer Settings Category and the MSP Specific Settings Category
+- Select the areas that require white labeling by either selecting Yes/No for show/hide features, or, entering a URL to Images or Links
+- If there are items that do not need to be changed, leave the default text of No Change and it will be ignored
+- Run the policy and review if there are any errors, an incident will be generated if an item fails and will tell you which Item has failed and why
+- Once run, you should terminate the policy so it does not continue to run
+
+## Input Parameters
+
+This policy has the following input parameters required when launching the policy.
+
+### Applies to whole policy (Optional)
+
+- *Email addresses* - A list of email addresses to notify

```

Comment:
Addresses should be capitalized. Make sure this is also the case in the policy template.

Reply:
Unresolved because issue was not addressed.